main.go

```diff
@@ -1337,6 +1337,7 @@ func main() {
 	llvmFeatures := flag.String("llvm-features", "", "comma separated LLVM features to enable")
 	cpuprofile := flag.String("cpuprofile", "", "cpuprofile output")
 	monitor := flag.Bool("monitor", false, "enable serial monitor")
+	baudrate := flag.Int("baudrate", 0, "baudrate of serial monitor")
```

Comment:
Wouldn't it be better to just use default value of `115200` here instead of this code?

https://github.com/tinygo-org/tinygo/pull/3190/files#diff-23669db2c16e34742ce163e9f6bc55e0e78d04ee8c1eb292f57b6967805fe12dR36-R38

Reply:
Indeed, 115200 bps seems to be the default here.

> https://github.com/tinygo-org/tinygo/pull/3190/files#diff-23669db2c16e34742ce163e9f6bc55e0e78d04ee8c1eb292f57b6967805fe12dR36-R38

I think it is safer to leave error handling in place.